dg/media/raw_data_analytics/raw_data_analytics.js

```diff
@@ -1,185 +1,236 @@
+        if (checked_values[i].checked) {
+            count_values++;
+        }
+    }
+
+    if ((count_partition == 0) && (count_values == 0) && (count_partition_restrict == 0) && (!list.checked)) {
+        alert("Please select some fields !!");
+        event.preventDefault();
+    }
+
+    else if ((count_partition != 0) && (count_values == 0) && (!list.checked)) {
+        alert("Please select atleast one value field!!");
+        event.preventDefault();
+
+    }
+
+
+    if (list.checked) {
+        for (i = 0; i < checked_partitions_restrict.length; i++) {
+            if (checked_partitions_restrict[i].checked) {
+                count_partition_restrict++;
+            }
         }
+        if (count_partition_restrict > 1) {
+            alert("Along with list please select either Animator/Group/Registered Viewers/Video from Partitions");
+            error = 1;
+            event.preventDefault();
 
-        else if((count_partition_restrict == 0) && (count_partition == 0)) 
-        {
-          alert("Select atleast one option from partition fields!!!");
-          error = 1;
-          event.preventDefault();
-         
         }
 
+        else if ((count_partition_restrict == 0) && (count_partition == 0)) {
+            alert("Select atleast one option from partition fields!!!");
+            error = 1;
+            event.preventDefault();
 
-     }
-     //alert(list_video.selectedIndex);
-     if( ((animator.checked) && (animator_no.checked))           || 
-         ((people.checked)   && (animator_no.checked))           ||
-         ((people.checked)   && (attendance.checked))            ||
-         ((group.checked)    && (animator_no.checked))           ||
-         ((video.checked)    && (animator_no.checked))           ||
-         ((video.checked)    && (video_screened_no.checked))     ||
-         ((video.checked)    && (video_produced_no.checked)))
-          {
-           alert("Invalid combination of 'Value' and 'Partition' fields!! Please check");
-           error = 1;
-           event.preventDefault(); 
-          }
-
-     if((video.checked) && (list.checked) && (list_video.selectedIndex == 0))
-     {
+        }
+
+
+    }
+    //alert(list_video.selectedIndex);
+    if (((animator.checked) && (animator_no.checked)) ||
+        ((people.checked) && (animator_no.checked)) ||
+        ((people.checked) && (attendance.checked)) ||
+        ((group.checked) && (animator_no.checked)) ||
+        ((video.checked) && (animator_no.checked)) ||
+        ((video.checked) && (video_screened_no.checked)) ||
+        ((video.checked) && (video_produced_no.checked))) {
+        alert("Invalid combination of 'Value' and 'Partition' fields!! Please check");
+        error = 1;
+        event.preventDefault();
+    }
+
+    if ((video.checked) && (list.checked) && (list_video.selectedIndex == 0)) {
         alert("Please select list of videos produced or list of videos produced from dropdown!!!");
```

Comment:
Change the alert message "Please select List of videos screened or list of videos produced from dropdown"